test/letters/containers/AddressSection.unit.spec.jsx

```diff
@@ -100,162 +97,161 @@ describe('<AddressSection>', () => {
       }
     };
 
-    const tree = SkinDeep.shallowRender(<AddressSection {...props}/>);
-    const addressBlock = tree.dive(['AddressContent', 'AddressBlock']);
-    const addressBlockText = addressBlock.subTree('.address-block', 'div.letters-address').text();
+    const tree = shallow(<AddressSection {...props}/>);
+    const addressBlock = tree.find('AddressContent').dive()
+      .find('AddressBlock').dive();
+    const addressBlockText = addressBlock.find('.address-block div.letters-address.street').text();
     expect(addressBlockText).to.contain('2476 main street, ste #12 west');
   });
 
   it('should render an edit button if user is allowed to edit address', () => {
-    const component = ReactTestUtils.renderIntoDocument(<AddressSection {...defaultProps}/>);
-    const editButton = ReactTestUtils.findRenderedDOMComponentWithTag(component, 'button');
-    expect(editButton).to.not.be.empty;
+    const component = shallow(<AddressSection {...defaultProps}/>);
+    const editButton = component.find('button');
+    expect(editButton).to.have.lengthOf(1);
```

Comment:
We could use the chai-enzyme library and something like https://github.com/producthunt/chai-enzyme#present.

Might be clearer.